### DISCUSSION TITLE
Show all attachment

### DISCUSSION THREAD

**Discussion by nk3d** [tutao/tutanota#2650] [Bugs]
The "show all" attachment appearing only on web app. This important feature missing on desktop and mobile app.
Also not all attachment is visible for some reason on the same mail i saw 2 on mail app 4on desktop and when i save a saw 8 files!

**Comment by jowlo** [comment]
We introduced a bug that disabled the "show all" button outside of the web app and the display of attachments in some cases. We fixed it ASAP after noticing.

Do you still have the same problem with a current version of the app?
